src/main/java/ch/wisv/areafiftylan/controller/TeamRestController.java

```diff
@@ -156,9 +155,8 @@ public Team getTeamById(@PathVariable Long teamId) {
             "and @currentUserServiceImpl.hasAnyTicket(#username)")
```

Comment:
Yes, any Team related things require the user to have a Ticket. You can add users after you've sent them the ticket.